flink-ai-flow/lib/notification_service/java/src/main/java/org/aiflow/notification/client/NotificationClient.java

```diff
@@ -58,13 +65,39 @@ public NotificationClient(
             Boolean enableHa,
             Integer listMemberIntervalMs,
             Integer retryIntervalMs,
-            Integer retryTimeoutMs) {
+            Integer retryTimeoutMs,
+            Properties properties)
```

Comment:
Why not use `Properties` not `boolean enableIdempotence`?

Reply:
I want to put all parameters in `properties` in the future, instead of adding parameter every time.